Source/GSXibKeyedUnarchiver.m

```diff
@@ -264,6 +326,9 @@ - (id) initForReadingWithData: (NSData*)data
   NSXMLParser *theParser;
   NSData *theData = data;
 
+  // Dictionary which contains custom class information for Gorm/IB.
+  _customClasses = [[NSMutableArray alloc] initWithCapacity: 10];
```

Comment:
As the comment already suggests this should be a dictionary keyed on the custom class name. That way we don't need a loop when getting a loop to find the matching definitions.

Reply:
This would mean changing it here and in Gorm.  I would like to leave this how it is for the time being.

Reply:
UGH... No time like the present.  It needs to be keyed on the customClass name and ID.   This way when the unarchiver is working in Gorm it can pick up the record based on the ID of the object being unarchived.

Reply:
I understand the first argument, although I don't share it, that this would require changes to Gorm too. But what do you mean by the second argument? Gorm just loops over all the elements in the collection. For that it is completely irrelevant whether the collection is an array or a dictionary.

Reply:
This is done.  Pushed this code.